# Test for trying to figure out [#1](https://github.com/lzyutada/MyPython/issues/1)
- i can call 'f2' in package 'sd2' from path 'sd1/sd1_1/ex.py', but cannot calling directly at path 'sd1/sd1_1/'. seems to have some explain from [Python相对导入:ValueError: attempted relative import beyond top-level package by wangjinyu124419](https://blog.csdn.net/u011519550/article/details/104732765)
    - i found no more information about import package from sibling folder, but it seems have some other way to work it out.
- PS: the folder structure in this example is similar to #1. 